web/themes/joinup/templates/rdf_entity/rdf-entity--solution--listing-item.html.twig

```diff
@@ -21,7 +21,7 @@
     </div>
     {% if view_mode == 'view_mode_featured' %}
       <div class="listing__field">
-        <p class="listing__teaser">{{ content.field_is_description|render|striptags('<a>')|raw }}</p>
+        <p class="listing__teaser">{{ content.field_is_description|render|striptags|raw }}</p>
```

Comment:
Last one.

Reply:
About all of these leftovers, I made a comment about it in jira:
"in several templates there are still conditions for featured view mode. But we already abandoned this approach, correct? Maybe we should remove this view mode completely?"
These striptags functions are only enabled for view_mode_featured. I didn't change striptags settings for this view mode in config, because I am not sure if it is still in use. Maybe it is better to remove this view mode completely.

Reply:
Ah true you are right, we still have the separate view mode. The view mode will be replaced, but we will still have to implement the featured functionality at some point. Ok you can ignore these comments then.